public/asteroids.js

```diff
@@ -257,6 +268,21 @@ Stage(function(stage) {
 
 Stage({
   textures : {
+    'bg' : Stage.canvas(function(ctx) {
```

Comment:
this should really not be a texture, but a physical object which lives in physics

Reply:
makes it to detect collisions, modify, etc...